dataclients/kubernetes/testdata/routegroups/convert/missing-service-1-of-2.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: myapp
+spec:
+  hosts:
+    - example.org
+  backends:
+    - name: myapp-1
+      type: service
+      serviceName: myapp-1
+      servicePort: 80
+    - name: myapp-2
+      type: service
+      serviceName: myapp-2
+      servicePort: 80
+  defaultBackends:
+    - backendName: myapp-2
+  routes:
+    - path: /app1
+      backends:
+        - backendname: myapp-1
+    - path: /app2
+      backends:
+        - backendname: myapp-2
+    - path: /split
+      backends:
+        - backendname: myapp-1
+        - backendname: myapp-2
```

Comment:
I would expect this to have a route for /split and myapp-2.

Reply:
Proposed change creates routes for services that exist but does not create routes if any of the backends is missing like before. The traffic split will not be correct if service is missing.

Reply:
Yes I see and it's an enhancement, but if you think about an actual deployment with a wrong config for the new version deployment (for example missing service.yaml for myapp-1), then this could lead in traffic switching cases to unexpected routing drops

Reply:
It is not worse than before and I want to fix the case we have. We may consider creating routes for the /split case separately

Reply:
We would need to refactor traffic calculation. Currently we discover missing services on the go but we'd need to first check all services, exclude missing and only then recalculate traffic. Besides there might be other reasons why it could not create a route for the routegroup (not allowed external backend or malformed predicate/filter) so I think its safe to just throw away routegroup route if there is any kind of error with it instead of trying to guess/construct half-working configuration.